### PR TITLE
fix(prometheus) fixed:

### PR DESCRIPTION
**Pull request by christianhilbrands**
- non existing host
- exclude testWebsites blackbox
- bumped chart

#### Checklist
- [ ] Functionality works & meets specifications (tested on rancher)
- [ ] Templates reviewed
- [ ] Added values to questions
- [ ] Bumped the chart version
- [ ] Updated appVersion (if needed)

### CODE REVIEW

File: charts/molgenis-prometheus/scripts/targetWebGet.sh [hunk 1]
```diff
@@ -83,7 +88,7 @@ printf '%s\n' "${acceptWebArray[@]}" >> accept-http-list.yml
 printf '%s\n' "${acceptTargetArray[@]}" >> accept-target-list.yml
 printf '%s\n' "${testWebArray[@]}" >> test-http-list.yml
 printf '%s\n' "${testTargetArray[@]}" >> test-target-list.yml
-echo $(kubectl --token=$TOKEN create configmap targets-configmap --from-file prod-target-list.yml --from-file accept-target-list.yml --from-file test-target-list.yml --from-file prod-http-list.yml --from-file accept-http-list.yml --from-file test-http-list.yml -o yaml --dry-run | kubectl --token=$TOKEN replace -f -)
```
Comment (erikschaberg) [inline charts/molgenis-prometheus/scripts/targetWebGet.sh:86]: do we still need to pass this test-target-list.yml?

and if we dont need themm do we need to generate them as yml file(s)?

Comment (christianhilbrands) [reply charts/molgenis-prometheus/scripts/targetWebGet.sh:86]: if we dont pass the test-target-list.yml to the new configmap of kubernetes then we dont have the node_exporter callback. so we need the yaml file. we dont get the requests from the blackbox exporter and we dont get notified if a test VM is down.

Comment (christianhilbrands) [reply charts/molgenis-prometheus/scripts/targetWebGet.sh:86]: it is the most easy way to save them in a yaml file

Comment (erikschaberg) [reply charts/molgenis-prometheus/scripts/targetWebGet.sh:86]: ok

**Pull request merged**